aldryn_newsblog/tests/test_views.py

```diff
@@ -388,17 +388,17 @@ def test_articles_count_by_tags(self):
         # create unpublished article to test that it is not counted
         self.create_tagged_articles(
             1, tags=(tag_names[0],), is_published=False)
-        tag_slug2 = self.create_tagged_articles(
-            3, tags=(tag_names[1],)).keys()[0]
-        tag_slug3 = self.create_tagged_articles(
-            5, tags=(tag_names[2],)).keys()[0]
+        tag_slug2 = list(self.create_tagged_articles(
+            3, tags=(tag_names[1],)).keys())[0]
+        tag_slug3 = list(self.create_tagged_articles(
+            5, tags=(tag_names[2],)).keys())[0]
         tags_expected = [
             (tag_slug3, 5),
             (tag_slug2, 3),
         ]
         tags = Article.objects.get_tags(
             request=None, namespace=self.app_config.namespace)
-        tags = map(lambda x: (x.slug, x.num_articles), tags)
+        tags = [tag for tag in map(lambda x: (x.slug, x.num_articles), tags)]
```

Comment:
I don't understand why you're doing this. Why mix comprehensions and map? You could just use comprehensions. And the outside comprehension doesn't actually do anything (or am I missing something?)